git-extra/PKGBUILD

```diff
@@ -111,7 +111,12 @@ build() {
   export CXX=${MINGW_CHOST}-g++
   export SRCDIR=$srcdir
   export BUILDDIR=build-${MINGW_CHOST}
+  export TMPDIR=tmp
   mkdir -p ${BUILDDIR}
+  mkdir -p ${TMPDIR}
+  sed -e "s|@@PACKAGE_NAME@@|${MINGW_PACKAGE_PREFIX}-${_realname}|g" \
+    <update-via-pacman.bat >"$TMPDIR"/update-via-pacman.bat ||
+    (echo "Could not generate update-via-pacman script" && exit 1)
```

Comment:
The canonical way to do this is to have a `update-via-pacman.bat.in` script and generate the `update-via-pacman.bat` version from it. Also, could we use `GIT_EXTRA_PACKAGE_NAME` instead of the less-descriptive `PACKAGE_NAME`?

Reply:
Makes sense - in https://github.com/git-for-windows/build-extra/pull/461/commits/6746dc5cb33a1b05cc423df1e36cf1058d48c6cf, I went with an approach to copy `update-via-pacman.bat` into the build folder instead of the `startdir`, as its contents will be different for every architecture. So we'll only have an `update-via-pacman.bat.in` in the `startdir` moving forward. WDYT?

Reply:
> So we'll only have an `update-via-pacman.bat.in` in the `startdir` moving forward. WDYT?

@dennisameling sounds good! TBH I hope that we can soon replace `update-via-pacman.bat` with the `.ps1` script from https://github.com/git-for-windows/git-sdk-arm64/pull/4 (meaning: we can simply delete it from `git-extra`, as the script will live directly in the respective `git-sdk-*` repository).